src/Build/Graph/ProjectGraph.cs

```diff
@@ -56,9 +56,11 @@ public sealed class ProjectGraph
 
         private readonly Lazy<IReadOnlyCollection<ProjectGraphNode>> _projectNodesTopologicallySorted;
 
-        private GraphBuilder.GraphEdges Edges { get; }
+        private readonly bool _isSolution;
 
-        internal GraphBuilder.GraphEdges TestOnly_Edges => Edges;
+        private readonly GraphBuilder.GraphEdges _edges;
```

Comment:
Unrelated minor change: Changed `Edges` from a property to a field since it was private.